.github/renovate.json5

```diff
@@ -173,6 +173,11 @@
       "versioning": "regex:^v(?<major>\\d{2})(?<minor>\\d{2})(?<patch>\\d{2})$",
       "matchPackageNames": ["tccr.io/truecharts/photoprism"],
     },
+    {
+      "matchDatasources": ["docker"],
+      "versioning": "regex:^v(?<major>\\d{4})-(?<minor>\\d{1})(?<patch>\\d{1})$",
+      "matchPackageNames": ["tccr.io/truecharts/rss-bridge"],
+    },
```

Comment:
Our tags are in format `v2022-06` 

I've made the year as major, first digit of month as minor and second digit as patch.
That's because afaik renovate needs reference for all there (major,minor,patch).

Otherwise we need to modify the tag creation to not trim the `-DAY` part

Reply:
Disregard previous statement

quote from renovate docs
```
major, minor, and patch: at least one of these must be provided. When determining whether a package has updates, these values will be compared in the standard semantic versioning fashion. If any of these fields are omitted, they will be treated as if they were 0 -- in this way, you can describe versioning schemes with up to three incrementing values.
```